sos/report/plugins/cups.py

```diff
@@ -41,4 +74,12 @@ def setup(self):
 
         self.add_journal(units="cups")
 
+        jid_arguments = self.get_jid_strings()
+
+        if not jid_arguments:
+            return
```

Comment:
Nack. Do not perform `return` in `setup()` manually. The loop immediately following this will simply not call execute anything if `jid_arguments` is empty.